app/models/service_provider.rb

```diff
@@ -4,6 +4,8 @@
 require 'identity_validations'
 
 class ServiceProvider < ApplicationRecord
+  self.ignored_columns = %w[irs_attempts_api_enabled]
```

Comment:
(Asking out of curiosity, not opposition) What is the benefit of adding this? Is this just the first step to dropping the column?

Reply:
>Is this just the first step to dropping the column?

Yep, exactly that.

Reply:
Will this cause problems with the ServiceProviderSeeder if it still receives the `irs_attempts_api_enabled` attribute?

Reply:
Hmm, that's a good question. I think I'll need to coordinate with the partnerships team to work to remove existing `irs_attempts_api_enabled` in partner application configurations, and we may need to keep this column in the meantime.

Reply:
I wouldn't necessarily think it'd be an issue to continue marking it as ignored, though I'll want to check to confirm.

Reply:
```
[2] pry(main)> s.update(irs_attempts_api_enabled: true)
ActiveModel::UnknownAttributeError: unknown attribute 'irs_attempts_api_enabled' for ServiceProvider.
```

Reply:
Thanks for checking. In that case, I'll revert the change to add `ignored_columns` for now. Updated in c867fb4.